sam/src/SckBase.cpp

```diff
@@ -750,6 +751,7 @@ void SckBase::saveConfig(bool defaults)
 			config.sensors[i].enabled = sensors[static_cast<SensorType>(i)].defaultEnabled;
 			config.sensors[i].everyNint = 1;
 		}
+		config.dimled = 1.0;
```

Comment:
Mabye we can set the default directly in the Configuration struct?
That way we dont forget to explicitly set it when used somewhere else. :-)

Reply:
Hi! So this was implemented finally in the `dev` branch. It will be merged into master in the next stable release: https://github.com/fablabbcn/smartcitizen-kit-21/commit/c743639f296118d82463d59e1267f61f51a1049d